iframes/identity/identity.js

```diff
@@ -0,0 +1,71 @@
+const env = process.env.NODE_ENV
+let targetingUrl
+
+switch (env) {
+  case 'staging':
+  case 'development':
+    targetingUrl = 'https://www.msmaster.qa.paypal.com/targeting/qraphql'
```

Comment:
Hrm, for development I can see a use case where I want to hit a local targeting server. Maybe worth having a localhost address for development?

Reply:
Yeah, that'd be nice to have going forward. I'll add an additional option for local development. If a url is specified it'll be used here. If not, it'll default to msmaster.

Reply:
The only catch is that you'll still need to set up an endpoint somewhere to serve the modified iframe that points to your local instance.